docs/galaxy/FAQ.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_label: 'FAQ'
+sidebar_position: 2
+---
+# FAQs
+
+### How is it used?
+Galaxy serves to be the central network point where all requests sent to Galaxy will be redirected to the respective places.
+
+### How do I install Galaxy?
+1. Clone the repository ``` $ git clone --recurse-submodules https://github.com/tamu-datathon-org/galaxy.git ```
+2. Install [Docker](https://docs.docker.com/get-docker/)
+3. Install [Docker_Compose](https://docs.docker.com/compose/install/)
+4. Enter this command `$ docker-compose up `
+
+### How do I create a new route?
+1. Go to the vercel.json file
+2. Either on the redirect or the rewrite array, copy paste the code below
+```
+{source: "/route", destination: "url"}
```

Comment:
Instead of copy paste, I would recommend saying something like "add the following object to the list `nameOfList`". Also describe the source and destination string patterns if they have any.